docs/public-api/notification/liquid-staking.mdx

```diff
@@ -1,43 +1,45 @@
- | notificationParams.stakes.flag          | mandatory   | Boolean | if the observation is made about staking movements in the blockchain                                |
- | notificationParams.stakes.value         | optional    | Float   | value from when the staking is taken in consideration                                               |
- | notificationParams.stakes.type          | optional    | String  | use [priceType](/public-api/ids) varible value from when the staking is taken in consideration      | 
- | notificationParams.queueUnbond.flag     | mandatory   | Boolean | if the observation is made about queueUnbond movements in the blockchain                            |
- | notificationParams.queueUnbond.value    | optional    | Float   | value from when the queueUnbond is taken in consideration                                           |
- | notificationParams.queueUnbond.type     | optional    | String  | use [priceType](/public-api/ids) varible value from when the queueUnbond is taken in consideration  |
- | notificationParams.submitBatch.flag     | mandatory   | Boolean | if the observation is made about submitBatch movements in the blockchain                            |
- | notificationParams.submitBatch.value    | optional    | Float   | value from when the submitBatch is taken in consideration                                           |
- | notificationParams.submitBatch.type     | optional    | String  | use [priceType](/public-api/ids) varible value from when the submitBatch is taken in consideration  |
- | notificationParams.finishUnbond.flag    | mandatory   | Boolean | if the observation is made about finishUnbond movements in the blockchain                           |
- | notificationParams.finishUnbond.value   | optional    | Float   | value from when the finishUnbond is taken in consideration                                          |
- | notificationParams.finishUnbond.type    | optional    | String  | use [priceType](/public-api/ids) varible value from when the finishUnbond is taken in consideration |
- | notificationParams.scope                | mandatory   | String  | use [scope](/public-api/ids) varible value                                                          |
+ | Parameter            | Requirement | Type    | Description                                                                                             |
+ | :------------------- | :---------- | :------ | :------------------------------------------------------------------------------------------------------ | 
+ | contract             | mandatory   | String  | Params of the notification                                                                              |
+ | tokenContract        | mandatory   | String  | Params of the notification                                                                              |
```

Comment:
Maybe change on contract and tokenContract the description `Params of the notification` to `Smart Contract Address` and `Bonded Token Address`, respectively.